wild/tests/sources/lto-undefined.c

```diff
@@ -0,0 +1,5 @@
+//#LinkArgs:-L./ -lexample
+
+extern int CRC32_ProcessSingleBuffer(void const*, int);
```

Comment:
Declaring an extern function won't actually cause a reference to the symbol to be emitted. The code needs to actually refer to the symbol - e.g. from main.